db/migrate/20150115155947_add_scaptimony_scap_content_digest.rb

```diff
@@ -0,0 +1,11 @@
+class AddScaptimonyScapContentDigest < ActiveRecord::Migration
+  def change
+    add_column :scaptimony_scap_contents, :digest, :string, :limit => 128
+    Scaptimony::ScapContent.find_each do |content|
```

Comment:
I know.

I would prefer to do it only after we have this problem.

We will notice quickly as new installation will fail.